shop-the-look/README.md

```diff
@@ -0,0 +1,382 @@
+- **Extensive Asset Library**: Demo app includes over 45,000 royalty-free images and videos from Pexels, pre-embedded and upserted for immediate use.
+- **Convenient Scripts**: Includes scripts to easily upload your own images and videos for your own use cases.
+
+## 🔧 Setup & Installation
+
+We offer two installation methods for Shop The Look:
+
+1. [Demo Deployment](#%EF%B8%8F-demo-deployment-2-minutes) - quick setup for demo purposes
+3. [Full Deployment](#%EF%B8%8F-full-deployment-30-minutes) - bring your own images and videos + full setup
+
+#### Not sure which installation method to use? 
+
+![Installation Flowchart](./assets/installation-flowchart.png)
+
+## ☁️ Demo Deployment (2 minutes)
+
+*Note: This is for **demonstration purposes only**.*
+
+For developers who want to quickly deploy and test the Shop The Look application without setting up their own backend services or supply their own image/video assets, we offer a demo deployment option that includes over 45,000 royalty-free images and videos. This method allows you to deploy only the front-end to Vercel or run it locally, while utilizing our hosted backend API (which we have set up with all assets, Pinecone Serverless index, and Google Cloud Vertex AI).
+
+### Benefits
+- No need to set up Pinecone or Google Cloud accounts
+- Contains over 45,000 royalty-free image and video assets embedded, upserted, and uploaded
+- Faster deployment process
+- For testing and exploration (not for production)
+
+### Prerequisites
+- Node.js (v14 or later)
+- npm or yarn
+- A Vercel account (for Vercel deployment)
+
+### 💻 Local Demo Deployment
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/pinecone-io/sample-apps-internal.git
+   cd sample-apps-internal/shop-the-look
+   ```
+
+2. Install frontend dependencies:
+   ```bash
+   npm install
+   # or
+   yarn install
+   ```
+
+3. Start the frontend server:
+   ```bash
+   npm run next-dev
+   # or
+   yarn next-dev
+   ```
+
+5. Open [http://localhost:3000](http://localhost:3000) in your browser.
+
+### 🚀 Vercel Demo Deployment
+
+TODO: npx create-next-app@latest --app https://github.com/pinecone-io/sample-apps-internal.git
+
+1. Clone the repository to your GitHub account.
+   ```bash
+   git clone https://github.com/pinecone-io/sample-apps-internal.git
+   cd sample-apps-internal/shop-the-look
+   ```
+
+2. Log in to your Vercel account and click "New Project".
+
+3. Import the repository.
+
+4. In the "Configure Project" step:
+   - Set the Framework Preset to Next.js
+   - In the "Environment Variables" page, modify `NEXT_PUBLIC_VERCEL_ENV` to `demo`:
+     ```
+     NEXT_PUBLIC_VERCEL_ENV=demo
+     ```
+
+5. Click "Deploy".
+
+Your Shop The Look front-end should now be deployed and accessible via the Vercel URL provided, using our hosted backend API for all data operations.
```

Comment:
Is this true? Are demo deployers all using the same hosted backend endpoint?

Reply:
Yup! This is for the fastest and easiest demo setup. Purely for testing and demo-ing. The demo deployers can also deploy a full app, which requires a full frontend and backend setup. 

The intent is, if the app blows up, we can turn demo-mode off. But I guess that is a good problem to have. 

When you run `npx create-pinecone-app`, it will deploy the demo app which will require zero setup on the user's end.